src/test/HostActivation.Tests/CommandExtensions.cs

```diff
@@ -36,6 +36,11 @@ public static Command EnableTracingAndCaptureOutputs(this Command command)
                 .CaptureStdErr();
         }
 
+        public static Command WithoutDotNetRoot(this Command command)
```

Comment:
Nit: I would prefer "positive" methods - in this case I think it would make sense to be able to simply call `.DotNetRoot(null)` (to unset the variable).